twoplayer-CHR-01.chr.ips.s

```diff
@@ -48,3 +48,27 @@ NEXT_ON_TOP = 0
         ips_tilehdr CHR_RIGHT,$9F
         .incbin "build/twoplayer.chrs/17"
 .endif
+
+.ifdef TOURNAMENT_MODE
```

Comment:
FYI: to debug a `make` error, you can use `make V=1` to enable output of the commands being run.

Reply:
I never had this error and was not aware that IPS even had that kind of limitation. But I remember I read somewhere in your notes something about sorting segments manually, is this reason of this?

Some research suggests, that flips might have changed that behaviour at some point in time: https://github.com/Alcaro/Flips/commit/8361a415f11033cd388e78315082a64611f099d2#diff-245a3c56044735826ef555d64e638eb9

Reply:
If you didn't see it, then yeah, the behavior changed. That would be quite convenient. I'm on flips 1.31 because that is the most recent release. I wish https://github.com/Alcaro/Flips would tag a release occasionally, but I can swap to it fairly trivially. Problem would probably be for Windows/Mac users that would want to use pre-built binaries.

The normal sorting of segments was in `.ips.cfg` files. The CHR patches are a bit simplified so it is sorting these lines instead of sorting segments.